git_flow.rst

```diff
@@ -0,0 +1,185 @@
+.. -*- mode: rst -*-
+
+.. role:: bash(code)
+   :language: bash
+
+Recommended GitHub workflow
+===========================
+Typically there are three types of people who use GitHub to access source code:
+users, developers, and maintainers. Users_ simply want access to the latest
```

Comment:
Are these links here to jump to the appropriate section? They don't work yet, but I'm guessing they will after being merged, right?